modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -136,6 +144,7 @@ ug_delete(Ids, Context) ->
 
     end.
 
+-spec ug_move_and_delete([pos_integer()], m_rsc:resource_id(), #context{}) -> ok.
```

Comment:
The spec in line 147 uses a record, please define a type for the record and use that instead.